utility/src/main/res/values/dimens.xml

```diff
@@ -3,7 +3,10 @@
   <dimen name="bullet_radius">4dp</dimen>
   <dimen name="bullet_gap_width">16dp</dimen>
   <dimen name="bullet_y_offset">2dp</dimen>
-  <dimen name="bullet_leading_margin">24dp</dimen>
+  <!--  Space between the start of the line and the bullet in a bullet span-->
```

Comment:
Nit
`<!--  Space between the start of the line and the bullet in a bullet span. -->`

Reply:
Done